app/src/main/java/com/sunasterisk/travelapp/data/models/Location.kt

```diff
@@ -7,6 +7,7 @@ import kotlinx.android.parcel.Parcelize
 import org.json.JSONArray
 import org.json.JSONException
 import org.json.JSONObject
+import kotlin.jvm.Throws
```

Comment:
sao lại thêm

Reply:
nhưng trước đó đâu có import đâu nhỉ 
https://viblo.asia/p/kotlin-coding-conventions-se-khac-voi-mac-dinh-cua-intelij-idea-nhu-the-nao-ByEZkOJqZQ0

Reply:
t xoá nó đi rồi